exasol_integration_test_docker_environment/test/test_test_environment_db_disk_size.py

```diff
@@ -40,15 +42,15 @@ def assert_disk_size(self, size: str):
     def test_default_db_disk_size(self):
         self.docker_environment_name = "test_default_db_disk_size"
         docker_envs = self.test_environment.spawn_docker_test_environment(self.docker_environment_name)
-        self.assert_disk_size("2 GiB")
-        close_environments(*docker_envs)
+        with ContextSpawnedTestEnvironment(*docker_envs):
```

Comment:
Usually, you try to generate the object for the context directly in the with statement